pymc/tests/test_distributions.py

```diff
@@ -3407,7 +3417,8 @@ def test_no_warning_logp(self):
         "sd_dist",
         [
             pm.Exponential.dist(1),
-            pm.MvNormal.dist(np.ones(3), np.eye(3)),
+            # For some reason this test is running when TimeSeries test is run
+            # pytest.mark.xfail(pm.MvNormal.dist(np.ones(3), np.eye(3)),)
```

Comment:
Weird. Is this locally, or also on the Github workflow?

Reply:
Not sure, I'll uncomment when PR is ready since it burns up a lot of CI time trying to figure out